scripts/konflux-build-triage/run_headless.sh

```
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Run konflux-triage headless (e.g. from cron).
+# Usage: run_headless.sh
+#
+
+set -euo pipefail
+
+# Source token from env file
+if [ -f "$HOME/.konflux-env" ]; then
+    source "$HOME/.konflux-env"
```

Shellcheck (0.11.0)

[info] Not following: ./.konflux-env was not specified as input (see shellcheck -x).

(SC1091)